app/Repositories/PostRepository.php

```diff
@@ -65,9 +65,6 @@ public function find($id)
     public function create(array $data, $signupId)
     {
         if (isset($data['file'])) {
-            // Auto-orient the photo by default based on exif data.
-            $image = Image::make($data['file'])->orientate();
```

Comment:
@sbsmith86 I think you still gotta make it because you use it below, but remove the call to `orientate()`

Reply:
@katiecrane yep! Refresh